### ISSUE TITLE
pyFaup ModuleNotFound error returning?

### ISSUE THREAD

**Issue by packetwatch** [stricaud/faup#102]
Hi all,

This feels like an instance of issue #85, but the fix is not working. It's basically a ModuleNotFound error for pyFaup. I installed through pip and also through the 'python setup.py install' route in /src/lib/bindings/python. 

Ubuntu 18 with MISP up to date. 

I'm trying to get mail_to_misp to run and failing on the test, 'mail_to_misp.py -h'. Results below:

server-name:/opt/mail_to_misp/faup/src/lib/bindings/python# python setup.py install
running install
running bdist_egg
running egg_info
writing pyfaup.egg-info/PKG-INFO
writing top-level names to pyfaup.egg-info/top_level.txt
writing dependency_links to pyfaup.egg-info/dependency_links.txt
reading manifest file 'pyfaup.egg-info/SOURCES.txt'
writing manifest file 'pyfaup.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/pyfaup
copying build/lib.linux-x86_64-2.7/pyfaup/bind.py -> build/bdist.linux-x86_64/egg/pyfaup
copying build/lib.linux-x86_64-2.7/pyfaup/__init__.py -> build/bdist.linux-x86_64/egg/pyfaup
copying build/lib.linux-x86_64-2.7/pyfaup/faup.py -> build/bdist.linux-x86_64/egg/pyfaup
copying build/lib.linux-x86_64-2.7/pyfaup/functions.py -> build/bdist.linux-x86_64/egg/pyfaup
byte-compiling build/bdist.linux-x86_64/egg/pyfaup/bind.py to bind.pyc
byte-compiling build/bdist.linux-x86_64/egg/pyfaup/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-x86_64/egg/pyfaup/faup.py to faup.pyc
byte-compiling build/bdist.linux-x86_64/egg/pyfaup/functions.py to functions.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying pyfaup.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying pyfaup.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying pyfaup.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying pyfaup.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist/pyfaup-1.0-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing pyfaup-1.0-py2.7.egg
Copying pyfaup-1.0-py2.7.egg to /usr/local/lib/python2.7/dist-packages
Removing pyfaup 1.0 from easy-install.pth file
Adding pyfaup 1.0 to easy-install.pth file

Installed /usr/local/lib/python2.7/dist-packages/pyfaup-1.0-py2.7.egg
Processing dependencies for pyfaup==1.0
Finished processing dependencies for pyfaup==1.0

Then:

server-name:/opt/mail_to_misp# ./mail_to_misp.py -h
Traceback (most recent call last):
  File "./mail_to_misp.py", line 12, in <module>
    from mail2misp import Mail2MISP
  File "/opt/mail_to_misp/mail2misp/__init__.py", line 3, in <module>
    from .mail2misp import Mail2MISP
  File "/opt/mail_to_misp/mail2misp/mail2misp.py", line 13, in <module>
    from pyfaup.faup import Faup
ModuleNotFoundError: No module named 'pyfaup'

It seems like the install succeeds but the module is not in a path maybe?

I also tried ldd /usr/local/bin/faup with no luck for the virtual env.

Any help is appreciated!

**Comment by packetwatch** [comment]
OK, for the sake of others, I found it and will document it here.

TLDR; 2 versions of Python installed. Mail_to_MISP wants python3. 


To fix it, I ran:

servername:/opt/mail_to_misp/faup/src/lib/bindings/python# python3 setup.py install

....Installed /usr/local/lib/python3.6/dist-packages/pyfaup-1.0-py3.6.egg
Processing dependencies for pyfaup==1.0
Finished processing dependencies for pyfaup==1.0

Additionally, for this app, I then needed to clone and install defang.

Clone defang through git, then:
servername:/opt/mail_to_misp/defang# python3 setup.py install

Afterwards, mail_to_misp worked.
